src/rest/QuestionAnswerResponse.ts

```diff
@@ -1,10 +1,17 @@
-export interface IQuestionAnswerResponse {
+export interface IPartialQuestionAnswerResponse {
   question: string;
   answerSnippet: string;
   documentId: {
     contentIdKey: string;
     contentIdValue: string;
   };
   score: number;
-  relatedQuestions: any[];
+  raw: { [key: string]: any } & {
+    title: string;
+    uri: string;
+  };
```

Comment:
Hmm, this intersection type looks unusual. Could it be simplified to just the second half?

```
raw: {
    title: string;
    uri: string;
};
```
We can also create a dedicated interface: `IQuestionAnswerRaw`.

Reply:
Actually, I'm not sure why I even added this. I'm going to remove it.